android/src/main/java/com/applovin/reactnative/AppLovinMAXAdViewManager.java

```diff
@@ -17,6 +18,8 @@
 class AppLovinMAXAdViewManager
         extends SimpleViewManager<AppLovinMAXAdView>
 {
+    private final static Map<String, AppLovinMAXAdView> mAdViews = new HashMap<>( 2 );
```

Comment:
We don't use `m` notation for Java fields. Since this is a final map, we should name it `ALL_AD_VIEWS` or something.

Reply:
Also the ordering of modifiers should be `private static final`.